Gemfile

```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
 
 gem 'hydra', git: 'https://github.com/projecthydra/hydra.git'
+gem 'active_annotations', git: 'https://github.com/avalonmediasystem/active_annotations.git', tag: 'avalon-r5'
```

Comment:
I was collecting our gem dependencies lower down in the Gemfile.  Could you please move these there?